src/app/Scenes/Home/Components/ArtworkRecommendationsRail.tsx

```diff
@@ -14,21 +15,27 @@ import { useTracking } from "react-tracking"
 import { RailScrollProps } from "./types"
 
 interface ArtworkRecommendationsRailProps {
-  title: string
-  me: ArtworkRecommendationsRail_me$key
+  isRailVisible?: boolean
   mb?: SpacingUnit
+  me: ArtworkRecommendationsRail_me$key
+  title: string
 }
 
 export const ArtworkRecommendationsRail: React.FC<
   ArtworkRecommendationsRailProps & RailScrollProps
-> = memo(({ title, me, scrollRef, mb }) => {
+> = memo(({ isRailVisible, mb, me, scrollRef, title }) => {
   const { trackEvent } = useTracking()
 
   const { artworkRecommendations } = useFragment(artworksFragment, me)
 
   const railRef = useRef<View>(null)
   const listRef = useRef<FlatList<any>>(null)
 
+  const { onViewableItemsChanged, viewabilityConfig } = useItemsImpressionsTracking({
+    isRailVisible: !!isRailVisible,
```

Comment:
nitpick: No need to convert to a boolean here:

```suggestion
    isRailVisible,
```